inc/class-styles.php

```diff
@@ -572,17 +575,18 @@ public function maybeUpdateStylesheets() {
 		// Theme was updated?
 		$theme = wp_get_theme();
 		$current_theme_version = $theme->get( 'Version' );
-		$last_theme_version = get_option( 'pressbooks_theme_version', $current_theme_version );
+		$last_theme_version = get_option( 'pressbooks_theme_version', 0 );
```

Comment:
Is this a  leftover from testing or on purpose?

Reply:
On purpose. I found that this was happening:

```
$current_theme_version = 1.1.0;
$last_theme_version = get_option( 'pressbooks_theme_version', $current_theme_version ); // If no option set, returns 1.1.0
```

Then `version_compare()` always shows that the theme hasn't been updated, never saves a version in the options table.